packages/features/users/components/UserTable/BulkActions/EventTypesList.tsx

```diff
@@ -36,7 +36,7 @@ export function EventTypesList({ table, orgTeams }: Props) {
   const { data } = trpc.viewer.eventTypes.getByViewer.useQuery({
     filters: { teamIds, schedulingType: [SchedulingType.ROUND_ROBIN] },
   });
-  const mutation = trpc.viewer.organizations.bulkAddToEventType.useMutation({
+  const mutation = trpc.viewer.organizations.bulkAddMembersToEventTypes.useMutation({
```

Comment:
Now that I think, I think the most straight forward name to this mutation would be `addMembersToEventTypes` without `bulk` in the name. And then also renaming `addBulkMembersToEventTypes.handler` to `addMembersToEventTypes.handler` - what do you think?

Reply:
Yeah makes sense

Reply:
While I am at it should I refactor for bulkDeleteUsers as well?

Reply:
I think it's best to create a new PR for "bulkDeleteUsers" as it falls out of the scope of this task : )